apps/avatax/bruno/1_Create_checkout_US.bru

```diff
@@ -0,0 +1,92 @@
+meta {
+  name: 1. Create checkout - US
+  type: graphql
+  seq: 2
+}
+
+post {
+  url: {{API_URL}}
+  body: graphql
+  auth: bearer
+}
+
+auth:bearer {
+  token: {{storefrontUserToken}}
+}
+
+body:graphql {
+  mutation CreateCheckout($channelSlug: String!, $variantId: ID!, $email: String!) {
+    checkoutCreate(
+      input: {
+        channel: $channelSlug
+        lines: [{ quantity: 10, variantId: $variantId }]
+        email: $email
+        shippingAddress:{
+           city: "New York",
+           country: US,
+           countryArea: "NY",
+           firstName: "John",
+           lastName: "Smith",
+           phone: "+12125094995",
+           postalCode: "10001",
+           streetAddress1: "8559 Lakes Avenue"
+        },
+        billingAddress:{
+           city: "New York",
+           country: US,
+           countryArea: "NY",
+           firstName: "John",
+           lastName: "Smith",
+           phone: "+12125094995",
+           postalCode: "10001",
+           streetAddress1: "8559 Lakes Avenue"
+        }
+        languageCode: EN_US
+      }
+    ) {
+      errors {
+        field
+        message
+        code
+        variants
+        lines
+        addressType
+      }
+      checkout {
+        id
+        totalPrice {
+          gross {
+            currency
+            amount
+          }
+          tax {
+            currency
+            amount
+          }
+        }
+        shippingMethods {
+          id
+          name
+          price {
+            currency
+            amount
+          }
+        }
+      }
+    }
+  }
+  
+}
+
+body:graphql:vars {
+  {
+      "variantId": "{{productVariantId}}",
+      "channelSlug": "{{channel_USD}}",
+      "email": "{{userEmail}}"
+  }
+}
+
+vars:post-response {
+  checkoutId: res.body.data.checkoutCreate.checkout.id
+  ~deliveryMethodId: res.body.data.checkoutCreate.checkout.shippingMethods[0].id
```

Comment:
**Question:** do we need it?

If this won't be extracted (checkbox in unchecked under vars tab):
![CleanShot 2024-02-19 at 08 22 25@2x](https://github.com/saleor/apps/assets/9116238/8b1a3bba-0abc-405a-85d0-dbffc5426742)